library/roomSchemas/com.novoda.downloadmanager.RoomAppDatabase/3.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
I assume this file is autogenerated right?

Reply:
Yep. It's the migration file.